### PR TITLE
Displaying vehicle checks on the office page - MES-3875

### PR DESCRIPTION
**Pull request by edwardwilson**
## Description
Showing vehicle checks in the office page
## Checklist

- [x] PR title includes the JIRA ticket number
- [x] Branch is rebased against the latest develop
- [x] Code has been tested manually
- [x] PR link added to JIRA ticket
- [ ] One review from each scrum team
- [ ] Squashed commit contains the JIRA ticket number

## Screenshots (optional)
image

### CODE REVIEW

**Review by rossfauldsbjss — APPROVED** [review]
looks good

File: src/pages/office/components/vehicle-checks/vehicle-checks.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+import { QuestionResult } from '@dvsa/mes-test-schema/categories/Common';
+import { CompetencyOutcome } from '../../../../shared/models/competency-outcome';
+
+@Component({
+  selector: 'vehicle-checks',
+  templateUrl: 'vehicle-checks.html',
+})
+export class VehicleChecksComponent {
+  @Input()
+    display: boolean;
```
Comment (szabikr) [inline src/pages/office/components/vehicle-checks/vehicle-checks.ts:11]: why these indentations?

File: src/pages/test-report/cat-be/components/vehicle-checks/vehicle-checks.scss [hunk 2]
```diff
@@ -1,4 +1,4 @@
-vehicle-checks {
+.test-report-vehicle-checks {
```
Comment (szabikr) [inline src/pages/test-report/cat-be/components/vehicle-checks/vehicle-checks.scss:1]: The css namespace should match the html selector

**Review by szabikr — APPROVED** [review]
Would recommend action on these comments below

**Pull request merged**